### PR TITLE
Added stretch goal to fundraising frontpage.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

File: djangoproject/templates/fundraising/index.html [hunk 1]
```diff
@@ -163,7 +164,7 @@ <h1 class="visuallyhidden">Support Django</h1>
 
         <h2>Who were the Django Fellows?</h2>
         <p>The two Fellows for the duration of the pilot were:</p>
-        <p><b>Tim Graham</b> - a member of the core team for several years. For
+        <p><strong>Tim Graham</strong> - a member of the core team for several years. For
```
Comment (timgraham) [inline djangoproject/templates/fundraising/index.html]: why not Berker too?

**Comment by timgraham** [comment]
LGTM otherwise.

File: djangoproject/templates/fundraising/index.html (hunk 1 above)
Comment (jezdez) [inline djangoproject/templates/fundraising/index.html]: Good question!

**Comment by jezdez** [comment]
Merged in 768b8a65ed9eb6f1cb2e0b67a749f1be1afa07b3

**Pull request closed without merging**